### PR TITLE
Fix rich file errors

### PR DESCRIPTION
**Pull request by LoboRich**
Adds three new files and edits rich file.

### CODE REVIEW

File: second_file [hunk 1]
```diff
@@ -0,0 +1 @@
+second
```
Comment (edos4) [inline second_file]: Insert the name of your crush.

File: third_file [hunk 2]
```diff
@@ -0,0 +1 @@
+third
```
Comment (edos4) [inline third_file]: Change this to "Let's eat."

File: second_file (hunk 1 above)
Comment (LoboRich) [reply second_file]: ok

**Comment by edos4** [comment]
Good job!

**Pull request merged**